command/rootchain/deploy/deploy.go

```diff
@@ -544,18 +545,23 @@ func deployContracts(outputter command.OutputFormatter, client *jsonrpc.Client,
 	}
 
 	if err := g.Wait(); err != nil {
-		outputter.WriteCommandResult(&helper.MessageResult{
-			Message: "[ROOTCHAIN - DEPLOY] Successfully deployed the following contracts\n"})
+		results := []command.CommandResult{}
```

Comment:
Why do you need this slice, couldn't you have used `commandResults` instead?

Reply:
I wanted to place a message at the beginning, but that can also be done without the results slice.

Reply:
Indeed 🙂